c2corg_ui/static/js/user.js

```diff
@@ -100,4 +100,12 @@ app.UserController.prototype.logout = function() {
 };
 
 
+/**
+ * @export
+ */
+app.UserController.prototype.hasEditRightsOuting = function(users) {
+  this.auth.hasEditRightsOuting(users);
```

Comment:
We decided with @asaunier to leave this as is for reasons : 

- there is no direct link between checking if an url is protected and showing or not the button. Protected-url-btn just checks if the user clicking is logged and if he can be redirected to the linked page, if not -> auth form. 
- Whereas auth.checkEditRights will decide whether or not the button will be displayed at all. There are documents that are collaborative, private or require special rights and all should be checked separately from the protected-url-btn.

It is true that when when checkEditrights = true, the button will show up and the protected-url-btn checking for the user authentication is useless, because he's already logged in. But I could, for example, log out in between and still have  access to the page, so it's better to double-check.